docs/source/guide/explanation/algorithms/classification/multi_class_classification.rst

```diff
@@ -211,14 +194,56 @@ In particular, the smaller the data, the greater the performance improvement can
 .. image:: ../../../../../utils/images/multi_cls_selfsl_performance_Food-101.png
   :width: 600
 
-You can use Self-supervised learning like the below command.
+To enable self-supervised training, the command below can be executed.
+Unlike other tasks, `--val-data-root` is not needed.
 
 .. code-block::
 
-  $ otx train {TEMPLATE} ... \
+  $ otx train otx/algorithms/classification/configs/efficientnet_b0_cls_incr/template.yaml \
+      --train-data-root=tests/assets/imagenet_dataset_class_incremental \
       params \
       --algo_backend.train_type=SELFSUPERVISED
 
+After self-supervised training, pretrained weights can be use for supervised (incremental) learning like the below command:
+
+.. code-block::
+
+  $ otx train otx/algorithms/classification/configs/efficientnet_b0_cls_incr/template.yaml \
+      --train-data-roots=tests/assets/imagenet_dataset_class_incremental \
+      --val-data-roots=tests/assets/imagenet_dataset_class_incremental \
+      --load-weights={PATH/PRETRAINED/WEIGHTS}
+
+*******************************
+Supervised Contrastive Learning
+*******************************
+
+To enhance the performance of the algorithm in case when we have a small number of data, `Supervised Contrastive Learning (SupCon) <https://arxiv.org/abs/2004.11362>`_ can be used.
+More specifically, we train a model with two heads: classification head with Influence-Balanced Loss and contrastive head with `Barlow Twins loss <https://arxiv.org/abs/2103.03230>`_.
+The below table shows how much performance SupCon improved compared with baseline performance on three baseline datasets with 10 samples per class: CIFAR10, Eurosat-10, and Food-101.
+
```

Comment:
How about adding sentence ?

"Note, SL stands for Supervised Learning"

Reply:
Done